WordPress/src/main/java/org/wordpress/android/ui/posts/PrepublishingViewModel.kt

```diff
@@ -64,6 +67,7 @@ class PrepublishingViewModel @Inject constructor(private val dispatcher: Dispatc
     }
 
     private fun navigateToScreen(prepublishingScreen: PrepublishingScreen) {
+        _dismissKeyboard.postValue(Event(Unit))
```

Comment:
```suggestion
        // if destintaion is HOME or TAGS and the current is HOME OR TAGS just don't dismiss the keyboard to avoid
        // re-layouts
        if (prepublishingScreen != TAGS && prepublishingScreen != HOME) {
            _dismissKeyboard.postValue(Event(Unit))
        }
```

I just tried this small tweak here to avoid the flickering that is observed when the keyboard is  up and we navigate to the Tags section: given we know both the HOME and the TAGS screens have an EditText, we can ask to send the dismissKeyboard signal only when we're not either in one of these nor navigating towards one of these. At this point in code we only know where we want to navigate to, but it's ok since landing on any of these two we'll want the keyboard to stay up if it was already up ;) (i.e. don't dismiss it).
For the case where this is not a story and hence there's no EditText in the HOME screen, we're ok too, because there wouldn't have been a keyboard up anyway.
Let me know your thoughts 👍

Reply:
Thanks for this @mzorz 
I had done this but I discovered this bug on my Google Pixel 4 XL, where sometimes the `KeyboardResizeUtil` doesn't get triggered.
Check this video out and let me know your thoughts. https://drive.google.com/file/d/1tb9lc0Yi_uzVKv9_XRdwoALFGnnW2xH9/view?usp=sharing

Reply:
Oh I see! ok let me check a bit and get back here